pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -525,6 +525,8 @@ let
         DRM_I915_GVT_KVMGT = module;
         # Enable Hyper-V Synthetic DRM Driver
         DRM_HYPERV = whenAtLeast "5.14" module;
+        # And disable the legacy framebuffer driver when we have the new one
+        FB_HYPERV = whenOlder "5.14" yes;
```

Comment:
How does setting it to yes sometimes disable it?

Reply:
OK just forcing it to no for newer ones and doing nothing for older ones does what we want.